injected/integration-test/test-pages/message-bridge/config/message-bridge-enabled.json

```diff
@@ -5,6 +5,10 @@
       "state": "enabled",
       "exceptions": []
     },
+    "favicon": {
```

Comment:
Do we need these changes here? It doesn't look like they're being tested are they?

It would be better to validate that "state": "disabled" actually disables the feature somewhere (if you're not already).

Reply:
it's a side-effect of our recent change. Without this, the favicon code is now running always - so I can either update the message-bridge tests, to expect additional messages, or disable here.

Reply:
And if the favicon code runs it breaks the test? Sorry just wasn't clear as a side effect.

Reply:
it only 'breaks' the test in the sense that I'd have to filter out the message. which feels even more wrong that doing this haha